kcptun.rb

```diff
@@ -0,0 +1,72 @@
+class Kcptun < Formula
+  desc "extremely simple & fast udp tunnel based on kcp protocol"
+  homepage "https://github.com/xtaci/kcptun"
+  url "https://github.com/xtaci/kcptun.git",
+    :tag => "v20160701", :revision => "13e980cdffb02e0b2bc793366e1314e663f549fc"
+
+  head "https://github.com/xtaci/kcptun.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath/".kcptun-gopath"
+    mkdir_p buildpath/".kcptun-gopath/src/github.com/xtaci"
+    cp_r cached_download, buildpath/".kcptun-gopath/src/github.com/xtaci/kcptun"
+    ENV.append_path "PATH", "#{ENV["GOPATH"]}/bin"
+    cd buildpath/".kcptun-gopath/src/github.com/xtaci/kcptun"
+    system "go", "get", "-v", "./client"
+    system "go", "build", "-ldflags", "-X main.VERSION=20160701 -s -w", "-o", "kcptun_client", "github.com/xtaci/kcptun/client"
+    bin.install "kcptun_client"
+    (bin/"kcptun").write <<-EOS.undent
+        #!/bin/sh
+        #{bin}/kcptun_client -r serverip:554 -l 127.0.0.1:1080 -mode fast2
+    EOS
+  end
+
+  def caveats; <<-EOS.undent
+    Now, you can use kcptun_client and kcptun (with config).
+    But if you want to use kcptun, you must modify it's config first.
+        vim #{opt_bin}/kcptun
```

Comment:
The program does not support `-c configfile` for now. I have to write a bash file to fill args, if want to use osx's plist.